### ISSUE TITLE
Groups owners or moderators can't see reports

### ISSUE THREAD

**Issue by ilMagnifico** [humhub/reportcontent#80]
The report module doesn’t work as advertised. Only the system administrator (currently me) receives the reports, not group owners or group moderators.

**Comment by marc-farre** [comment]
@ilMagnifico You need to go in the settings of a user Group and add users here:
![image](https://github.com/humhub/reportcontent/assets/23310825/8f5c93cb-9aae-4b70-8a7e-adcedf40000f)

**Comment by marc-farre** [comment]
For non-system admin users, they need to be an owner, admin or moderator of the space.
Then, they'll be notified about reported content from this space.

However, there is an exception:
> If we report a space admin post, we create a system admin only report (only visible in admin area)
See https://github.com/humhub/reportcontent/blob/master/models/ReportContent.php#L97

@luke- I think this adds confusion, mainly when testing the module to understand how it works.
Shouldn't we remove it?

**Comment by ilMagnifico** [comment]
I've done lots of tests, including deleting the module and re-installing it. I've reported things as admin, as a regular member and both in spaces of my own and other spaces. 
Even when I go to the "Reported content" tab which pops up for space owners when the reporting module is activated, there is nothing there on that tab. It just says that things will show up here if someone reports something.

**Comment by Semir1212** [comment]
@marc-farre Yes, as long as the sys admins are also notifed and the space admins can not remove the report by themselves.

**Comment by ArchBlood** [comment]
Currently testing for this issue;

1. Global content i.e. posts made directly on profile streams can't be viewed by those that aren't a system admin if reported, I believe this is because of Manage Module permissions, but even if you do have this permission you'll get "You are not permitted to access this section." error message because you also need the Manage Settings permissions as well;
2. Space Moderators & Owners can view and manage reported content within the Space stream and on the dashboard as long as the permissions are met;

The only issues that I'm seeing is no notifications are sent to owner/administrators or moderators and both Space & Global reports aren't filtered so whatever is reported in a Space also is sent to Global but I'd also like to note that point 1 also can be considered an issue as well here.

My idea when it comes to the logic that needs done here is as followed;

- Global Content: i.e. profile stream and/or other stream content reported should be visible in the Global Reported Content area only accessible by System Admin or those within the Administrators user group
- Space Content: i.e. space stream and/or other space module stream content reported should be visible in the Space Reported Content area only while also allowing for system admins to be able to view and manage as normal.

**Comment by ilMagnifico** [comment]
Again, this is the situation:
<img width="330" alt="Screenshot 2024-09-04 alle 19 55 01" src="https://github.com/user-attachments/assets/453f7cb2-b56b-44f5-9f35-2b290282e881">
I'm a mod of the space A.

People reports 2 different posts of that spaces of two different users with not any kind of permits, so normal users, just for test.
<img width="212" alt="Screenshot 2024-09-04 alle 19 54 42" src="https://github.com/user-attachments/assets/922d0e1d-526e-4e16-90ff-2d53e07cb836">
I can see them on the Admin page
<img width="892" alt="Screenshot 2024-09-04 alle 19 32 22" src="https://github.com/user-attachments/assets/75ccefbc-c1cc-4afc-9784-42faff22e9bd">
But when I try to have a look at them it doesn't work.

This must be sure a bug. I mean otherwise what's the point of being a Mod of a Space if then you can even see reports from users on that space?

**Comment by ilMagnifico** [comment]
bump @marjanapesic

**Comment by ilMagnifico** [comment]
@ArchBlood I'll get back to you when I download the 1.17.
